.github/workflows/ci-new.yml

```diff
@@ -36,7 +36,7 @@ env:
   PUBLIC_KEY_SERVICE_NAME: 'jvs-public-key-f22e'
   UI_SERVICE_NAME: 'jvs-ui-8ab7'
   SERVICES_URL_POSTFIX: '2nhpyabgtq-uc.a.run.app'
-  KMS_KEYRING_ID: 'jvs-keyring-0666'
+  KMS_KEYRING_ID: 'jvs-keyring-ci-${{ github.run_id }}-${{ github.run_number }}'
```

Comment:
Just curious, if this workflow fails and you click re-run, do you expect this number to be different? If yes, then you should use `github.run_attempt` instead of `run_number`.

Otherwise, LGTM